src/back/loader/Base.js

```diff
@@ -72,7 +72,9 @@ BaseLoader.prototype.normalizeStylesheet = function (style) {
 
 BaseLoader.prototype.load = function () {
     this.mml = this.loadFile();
-    this.postprocess();
+    if (this.project.config.parsed_opts.renderer == 'carto') {
```

Comment:
That would be better, but as far as I understand the Loader is independent from the renderer classes, so I would have to move all the logic to the Carto class. The processing would then need to take place just before rendering? I'm not sure if the processing of the MML really belongs into the renderer class.